confirmation/models.py

```diff
@@ -73,7 +72,10 @@ def get_object_from_key(
 
 
 def create_confirmation_link(
-    obj: ContentType, confirmation_type: int, url_args: Mapping[str, str] = {}
+    obj: ContentType,
+    confirmation_type: int,
+    validity_in_days: Optional[int] = None,
+    url_args: Mapping[str, str] = {},
```

Comment:
I mention this elsewhere, but we should use the `, *, ` technique to make `validity_in_days` a kwargs-only parameter for readability.

Reply:
(And maybe it's worth adding a comment that the parameter is an override for the default for the link type, and is primarily intended to be used in tests).